# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/gudev/hotfix-udev.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libgudev-237/meson.build.vanilla	2022-12-12 12:01:34.440479539 +0100
+++ libgudev-237/meson.build	2022-12-12 12:02:02.328480699 +0100
@@ -41,7 +41,7 @@
 cc = meson.get_compiler('c')
 
 glib_req = '>= 2.38.0'
-libudev_req = '>= 199'
+libudev_req = '>= 169'
 introspection_req = '>= 1.31.1'
 vapigen_req = '>= 0.38.0'
 gtk_doc_req = '>= 1.18'
